assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -31,32 +34,177 @@ export const drawDateRangeControls = function(elem, store, siteno) {
         period: 'custom',
         ariaExpanded: false
     }];
+    const CUSTOM_TIMEFRAME_RADIO_BUTTON_DETAILS = [
+        {
+            id: 'custom-input-days-before-today',
+            value: 'days',
+            text: 'days before today',
+            checked: true,
+            ariaExpanded: true
```

Comment:
I actually went round and round with this one. I didn't know the best route, since the 'days before today' is hidden, but it open when hidden. So I changed it back and forth a couple of times. But I'm happy to go with false.